app/ui/lib/SideModal.tsx

```diff
@@ -81,40 +80,39 @@ export function SideModal({
                 <DialogOverlay />
                 <AnimatedDialogContent
                   className="DialogContent ox-side-modal pointer-events-auto fixed bottom-0 right-0 top-0 z-sideModal m-0 flex w-[32rem] flex-col justify-between border-l p-0 bg-raise border-secondary elevation-2"
-                  aria-labelledby={titleId}
                   style={{
                     transform: x.to((value) => `translate3d(${value}%, 0px, 0px)`),
                   }}
+                  // shuts off a warning from radix about dialog content needing a description
+                  aria-describedby={undefined}
                 >
-                  {title && (
-                    <Dialog.Title asChild>
-                      <>
-                        <SideModal.Title id={titleId} title={title} subtitle={subtitle} />
-
-                        {errors && errors.length > 0 && (
-                          <div className="mb-6">
-                            <Message
-                              variant="error"
-                              content={
-                                errors.length === 1 ? (
-                                  errors[0]
-                                ) : (
-                                  <>
-                                    <div>{errors.length} issues:</div>
-                                    <ul className="ml-4 list-disc">
-                                      {errors.map((error, idx) => (
-                                        <li key={idx}>{error}</li>
-                                      ))}
-                                    </ul>
-                                  </>
-                                )
-                              }
-                              title={errors.length > 1 ? 'Errors' : 'Error'}
-                            />
-                          </div>
-                        )}
-                      </>
+                  <div className="items-top mb-4 mt-8">
+                    <Dialog.Title className="flex w-full items-center justify-between break-words pr-8 text-sans-2xl">
```

Comment:
`Dialog.Title` without `asChild` renders as an `h2`. Putting it directly around the title itself makes a lot more sense here.